exo-run/features/steps/steps-when.ls

```diff
@@ -29,6 +29,15 @@ module.exports = ->
       ..on 'ended', -> done!
 
 
+  @When /^running a "([^"]*)" application$/ timeout: 600_000, (@app-name, done) ->
+    @checkout-app @app-name
+    @app-dir := path.join process.cwd!, 'tmp', @app-name
+    @setup-app @app-dir, ~>
+      command = \exo-run
+      if process.platform is 'win32' then command += '.cmd'
+      @run-app {command}, done
```

Comment:
This functions looks to be a near exact copy of `Given a running "crashing-service" application` aside from the online-text. Could that be dried up?